lib/i18n/sfDateFormat.class.php

```diff
@@ -239,9 +239,10 @@ public function format($time, $pattern = 'F', $inputPattern = null, $charset = '
       }
       else
       {
-        $function = ucfirst($this->getFunctionName($pattern));
+        $function = $this->getFunctionName($pattern);
         if ($function != null)
```

Comment:
This syntax is not my cup of tea, because it's a bit confusing for begginer (assignation + compare on a same line)
I'd prefer to add parenthesis arround the assignation to make it more readable.

But I have checked, and symfony1 use this syntax, so let's go 👍